geode-core/src/main/java/org/apache/geode/internal/cache/RegionPerfStats.java

```diff
@@ -519,9 +519,15 @@ public void incEvictWorkTime(long delta) {
     cachePerfStats.incEvictWorkTime(delta);
   }
 
+  @Override
+  public void incRegionClearCount() {
+    stats.incLong(regionClearsId, 1L);
+    cachePerfStats.incClearCount();
+  }
+
   @Override
   public void incClearCount() {
-    stats.incLong(clearsId, 1L);
+    stats.incLong(bucketClearsId, 1L);
```

Comment:
I think this may possibly be incorrect. `incRegionClearCount()` is being overridden from `RegionStats` but every other method in this class is overriding methods from `CachePerfStats`. I think the correct methods might be `incClearCount()` which calls `cachePerfStats.incClearCount()` and `incBucketClearCount()` which calls `cachePerfStats.incBucketClearCount()`.

Reply:
It took me awhile to understand what you meant here - but you were correct. There was some weird inheretance going on here stemming from the fact that I didn't understand the relationship between RegionStats, RegionPerfStats, and CachePerfStats. These should be fixed now.